changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.7.0-beta3": [
+      "[Improved] Add support for RStudio Editor on MacOS - #11690. Thanks @geophilusd!",
+      "[Improved] Use system theme as default - #9959. Thanks @say25!",
```

Should we say "color theme" instead? Or not needed?
```suggestion
      "[Improved] Use system color theme as default - #9959. Thanks @say25!",
```

---

Hmm.. I have always heard it referred to as system theme because it technically impacts styles which is not necessarily just color tho that is our implementation. (For example, low vision theme I have seen on some apps also increases font-size)

---

Yep, I'd prefer to just stick to system theme.